desk/app/diary.hoon

```diff
@@ -85,7 +85,8 @@
     (mule |.(!<([versioned-state epic:e] vase)))
   =/  [old=versioned-state cool=epic:e]
     ?.  ?=(%| -.maybe-old)  p.maybe-old
-    [!<(versioned-state vase) okay]
+    =-  ~&  >  %bad-load  (fall - [state.cor okay])
```

Comment:
nukes state 😈

it will only do this if the old state does not correspond to any version we know about - i.e if you update `state-0` yourself. we wouldn't be able to migrate that state anyway. we could crash here too, but then you'd have to either nuke manually or write a custom migration to get it running. also with #1065 you'll never fully lose state anyway.